orchestrator/lib/coolfhir/proxy.go

```diff
@@ -83,7 +84,7 @@ func (f fhirClientProxy) ServeHTTP(httpResponseWriter http.ResponseWriter, reque
 			}, "FHIR request", httpResponseWriter)
 			return
 		}
-		if len(requestData) > 0 {
+		if len(requestData) > 0 && !strings.HasSuffix(request.URL.Path, "/_search") {
```

Comment:
They do, but we use it differently. If you go to line 129 you can see here:
```
if strings.HasSuffix(request.URL.Path, "/_search") {
			values, err := url.ParseQuery(string(requestData))
			if err == nil {
				err = f.client.SearchWithContext(request.Context(), strings.TrimSuffix(outRequestUrl.Path, "/_search"), values, &responseResource, params...)
			}
		}
```

This is just to make sure we don't try to marshall it to JSON because it is not in JSON format to begin with